hosting/src/firebase.ts

```diff
@@ -1,11 +1,15 @@
-import { getApp } from "firebase/app";
-import { useRouter } from "next/router";
+import { initializeApp } from "firebase/app";
+import { getFirestore } from "firebase/firestore";
 
-const firebaseAppName = useRouter().query.__firebaseAppName;
-const appName = Array.isArray(firebaseAppName)
-  ? firebaseAppName[0]
-  : firebaseAppName;
+const firebaseConfig = {
```

Comment:
We should put this data in .env.local file. Check [here](https://github.com/oddbit/tanam/pull/346/files#diff-1ccf3f8b27b5058bbe27a7070beca0b80735711fd493d6aab794fe7a4a57e5bd) for example